lib/src/timetable.dart

```diff
@@ -90,13 +95,38 @@ class _TimetableState extends State<Timetable> {
   @override
   void initState() {
     super.initState();
+
     _scrollController =
         widget.scrollController ?? ScrollController(initialScrollOffset: 0);
     if (widget.timeBlocks.isNotEmpty) {
       _scrollToFirstBlock();
     } else {
       _scrollToInitialTime();
     }
+
```

Comment:
115 & 120 seem magic? Those numbers should be configurable. With an assert to prevent those numbers being to close together resulting into an infinite call of onUnder-/OverScroll